cmd/bridge/main.go

```diff
@@ -503,6 +512,29 @@ func main() {
 		bridge.FlagFatalf("listen", "scheme must be one of: http, https")
 	}
 
+	var helmChartRepoProxyRootCAs *x509.CertPool
+	if *fHelmChartRepoUrlCaFile != "" {
+		helmChartRepoProxyRootCAs = x509.NewCertPool()
+		helmChartRepoProxyCertPEM, err := ioutil.ReadFile(*fHelmChartRepoUrlCaFile)
+		if err != nil {
+			log.Fatalf("failed to read helm chart repo ca file %v : %v", *fHelmChartRepoUrlCaFile, err)
+		}
+		if !helmChartRepoProxyRootCAs.AppendCertsFromPEM(helmChartRepoProxyCertPEM) {
+			log.Fatalf("No CA found for Helm chart repo proxy")
+		}
+	} else {
+		helmChartRepoProxyRootCAs, _ = x509.SystemCertPool()
```

Comment:
@spadgett is it ok to use system root ca (not sure what it is in the console pod) if Helm chart repo CA is not specified, and an external URL should be accessed? Should we have any worries in disconnected cluster mode, in particular when Helm chart repo uses a self-signed certificate?

Reply:
Yes, this is fine. It will be set to the trusted CA bundle by the console operator.

If the trusted CA bundle doesn't work, we give the option to set a CA file.

Reply:
As above, can this logic live under `pkg/helm` & be called via a func? 

```go
srv.HelmChartRepoProxyConfig = helm.chartRepoProxyConfig(args)
```
`main.go` is pretty messy.  Lets not let it grow if we can help it, much easier to put the logic in the right place now.